spec/rollbar/scrubbers/params_spec.rb

```diff
@@ -327,8 +327,8 @@
         let(:result) do
           [
             {
-              :foo => /\*+/,
-              :secret => 'the-secret',
+              :foo => 'bar',
+              :secret => /\*+/,
               :password => 'the-password',
               :password_confirmation => 'the-password'
```

Comment:
I would expect a whitelist to only allow exact matches. Not substring/fuzzy matches.
So `scrub_whitelist = [:password]` should result in `:password_confirmation` being scrubbed.

Reply:
Yeah that makes sense, it is a byproduct of the current way scrub_fields end up doing substring matches. The fact is if you had `password_confirmation` in the scrub_fields and put `password` in the whitelist then `password_confirmation` would get scrubbed. If you only have `password` in the scrub_fields, then both `password` and `password_confirmation` end up getting scrubbed due to the substring match, but then neither get scrubbed if you whitelist `password`.